docs/quickstart.rst

```diff
@@ -1,116 +1,81 @@
-
-    client = discord.Client(intents=discord.Intents.all())
-    slash = SlashCommand(client, auto_register=True)
-
-    @client.event
-    async def on_ready():
-        print("Ready!")
-
-    client.run("your_bot_token_here")
-
-1. 1st and 2nd line is importing library and extension.
-2. Then, creates client instance and extension instance.
-3. Finally, runs our bot.
-
-Well, now this will run but do literally nothing except printing ``Ready!``.
-
-Let's add one simple command.
-
-Fix your source code like this:
+For this example, ``main.py`` will be used.
 
 .. code-block:: python
 
     import discord
-    from discord_slash import SlashCommand
+    from discord_slash import SlashCommand # Importing the newly installed library.
 
     client = discord.Client(intents=discord.Intents.all())
-    slash = SlashCommand(client, auto_register=True)
-
-    guild_ids = [789032594456576001] # Put your testing server ID.
+    slash = SlashCommand(client, auto_register=True) # Declares slash commands through the client.
 
     @client.event
     async def on_ready():
         print("Ready!")
 
-    @slash.slash(name="ping", guild_ids=guild_ids)
-    async def _ping(ctx):
-        await ctx.send(content=f"Pong! ({client.latency*1000}ms)")
-
     client.run("your_bot_token_here")
 
-I'll explain what is changed:
-
-- ``guild_ids = [789032594456576001]``: This is for adding your command as a guild command. Otherwise, you need to wait for an hour to wait until your command is added.
-- ``@slash.slash(name="ping"...)`` ~ ``await ctx.send(...)``: This adds new slash command. This command basically sends an API latency.
+Let's give this a run. When you run this code, you'll see... nothing but ``Ready!``.
 
-Let's do some more complicated things:
+That's completely normal, because we haven't defined any slash commands yet.
+We can do so by adding this code shown here:
 
 .. code-block:: python
 
     import discord
-    from discord_slash import SlashCommand
-    from discord_slash import SlashCommandOptionType
-    from discord_slash.utils import manage_commands
+    from discord_slash import SlashCommand # Importing the newly installed library.
 
     client = discord.Client(intents=discord.Intents.all())
-    slash = SlashCommand(client, auto_register=True)
+    slash = SlashCommand(client, auto_register=True) # Declares slash commands through the client.
 
-    guild_ids = [789032594456576001] # Put your testing server ID.
+    guild_ids = [789032594456576001] # Put your server ID in this array.
 
     @client.event
     async def on_ready():
         print("Ready!")
 
     @slash.slash(name="ping", guild_ids=guild_ids)
-    async def _ping(ctx):
+    async def _ping(ctx): # Defines a new "context" (ctx) command called "ping."
         await ctx.send(content=f"Pong! ({client.latency*1000}ms)")
 
-    @slash.slash(name="echo", guild_ids=guild_ids, options=[manage_commands.create_option("string", "A random string.", SlashCommandOptionType.STRING, True)])
```

Comment:
Yes, quickstart should be minimal. But, in my opinion command with args should also be considered as essential.
(I might change my mind so just adding as a comment.)

Reply:
I think later on it's overall easier to describe how the options work under a part of the docs explaining the table structure of it since a lot of slash commands are built off of a JSON-supported model from Discord when HTTP req is sent. I'm not against it being added, but there should be some context behind it.

Reply:
Right. I think just creating another tutorial with advanced usage seems better.